Doc/Tutorial/chapter_uniprot.tex

```diff
@@ -25,15 +25,15 @@ \subsection{Parsing Swiss-Prot records}
 >>> from urllib.request import urlopen # Python 3 only
 >>> from io import TextIOWrapper
 >>> 
->>> handle = TextIOWrapper(urlopen("http://example.org/myprositefile.dat"))
+>>> handle = TextIOWrapper(urlopen("https://raw.githubusercontent.com/biopython/biopython/master/Tests/SwissProt/F2CXE6.txt"))
```

Comment:
That's a SwissProt text file, for a Prosite file maybe use ``Tests/Prosite/ps00107.txt``